include/openssl/e_os2.h

```diff
@@ -258,29 +258,6 @@ typedef unsigned __int64 uint64_t;
 #  include <stdint.h>
```

Comment:
some lines above is this apparently wrong define:
`# define PRIu64 "%Lu"`

should this also move?